.vib/pinniped/ginkgo/scripts/pinniped-auth.sh

```diff
@@ -4,7 +4,7 @@
 
 # Install required packages and download pinniped-cli
 install_packages -qq curl
-curl -JLks https://get.pinniped.dev/v0.20.0/pinniped-cli-linux-amd64 -o pinniped && chmod +x pinniped
+curl -JLks https://get.pinniped.dev/v0.24.0/pinniped-cli-linux-amd64 -o pinniped && chmod +x pinniped
```

Comment:
Would it be possible to parameterise this version? If not, this issue will repeat itself in the future and imo at least we should add a comment.

Reply:
I have added a comment because I don't know how to know the version.